microsoft-edge/devtools/performance/index.md

```diff
@@ -35,63 +35,119 @@ See also:
 
    ![CPU throttle](./index-images/capture-settings.png)
 
+   Or, in the **Environment settings** section in the lower right, in the **CPU throttling** dropdown menu, select **4x slowdown - recommended**.  The two instances of the dropdown menu are linked.
+
+   DevTools throttles your CPU so that it's 4 times slower than usual.  For example, the DevTools UI becomes sluggish when using these menus.
+
    A warning icon is displayed on the **Performance** tool's tab, to remind you that throttling is enabled.
 
 If you want to ensure that pages work well on low-end mobile devices, set **CPU** to **6x slowdown**.
 
+Continue with the steps below.
 
-<!-- ------------------------------ -->
-#### Set up the demo
 
-In the following section, continuing from above, you customize the demo to make sure that your experience is relatively consistent with the screenshots and descriptions.
+<!-- ====================================================================== -->
+## Use the demo's UI to compare Slow vs. Optimized code with few or many animated icons
 
-1. Click the **Add elements** button until the blue icons move noticeably slower than before.  On a high-end machine, you can click it about 20 times.
+In the Sluggish Animation webpage, first we'll show _few_ icons, and compare selecting the demo's **Slow** option button vs. the **Optimized** option button, producing _little_ difference in the speed and smoothness of animation.  Then we'll show _many_ icons, and compare selecting the demo's **Slow** option button vs. the **Optimized** option button, producing a _great_ difference in the speed and smoothness of animation.
 
-1. Click **Optimized**.  The blue icons should move faster and more smoothly.
+Use the demo's UI, as follows:
 
-1. To better display a difference between the optimized and unoptimized versions, click the **Remove elements** button a few times and try again.  If you add too many blue icons, you could max out the CPU, and then you might not observe a major difference in the results for the two versions.
+1. Switch to the Sluggish Animation webpage.
 
-1. Click **Slow**.  The blue icons move slower and with more sluggishness again.
+   **Show few icons, and compare Slow vs. Optimized**
 
+   There are 10 blue icons by default.  The **Slow** option button is selected by default.
 
-<!-- ------------------------------ -->
-#### Record runtime performance
+1. Watch a specific blue icon move up and down.
 
-When you ran the optimized version of the page, the blue icons move faster.  Why is that?  Both versions are supposed to move the icons the same amount of space in the same amount of time.  Take a recording in the **Performance** tool to learn how to detect the performance bottleneck in the unoptimized version.
+1. Select the **Optimized** option button.
 
-Continuing from above:
+   The 10 blue icons move a little more quickly and smoothly.
```

Comment:
This is highly dependent on your device. On mine, there's really no noticeable difference when there are only 10 icons. Both Slow and Optimized versions perform the same.

So maybe rephrase to:

```suggestion
   The 10 blue icons might move a little more quickly and smoothly, depending on your device.
```

OR:

Just skip the preliminary steps of comparing the performance of the Slow and Optimized settings, and instead ask the user to add icons until they notice that the performance visibly degrades.

Reply:
* i force-promoted all headings to h2
* i reworded headings for clarity
* below each heading, i put the hidden comment upstream anchor link - to facil. sync/Maint. 
in
https://learn.microsoft.com/microsoft-edge/devtools/performance/
~=
https://developer.chrome.com/docs/devtools/performance

@captainbrosset

Reply:
closing.  might open new Comment at this "todo"